src/cpp/r/CMakeLists.txt

```diff
@@ -15,6 +15,18 @@
 
 project (R)
 
+# find the version of R in play
+find_package(LibR REQUIRED)
+execute_process(
```

Comment:
Can we check for / report an error here if one occurs? (Basically signal for future-us that --slave is really gone now)

Reply:
This will already happen elsewhere:

https://github.com/rstudio/rstudio/blob/b22f63ee112c834842ec2d0ac442946d0e4c735c/src/cpp/CMakeLists.txt#L469-L478

If you're wondering why we don't piggyback on this check, it's because the `r/` folder is a separate CMake project which has no visibility of the parent project's variables.